webapp/src/app/shared/phaser/tilemap/tile-drawer.ts

```diff
@@ -156,6 +158,12 @@ export class TileDrawer extends BaseObject {
 			if (this.lastDraw.x === p.x && this.lastDraw.y === p.y) {
 				return;
 			}
+
+			// skip drawing if panning key is down
+			if (this.panKey.isDown) {
+				return;
+			}
+
```

Comment:
This clashes with the existing shift key functionality. Before this PR the shift key would allow one to draw without the auto tile feature interfering. I could see the old shift key functionality being rebound to CTRL.

Reply:
@2767mr Since this is a new feature,  would updated the old shift break users who are used to this feature?  I can update the panKey to `CTRL`, not a deal breaker.  I went with SHIFT initially because it felt more natural.

Reply:
It would not be a big deal because nobody knows it exists 😄

Reply:
Sounds good,  I'll push an update

Reply:
Done